pkg/controller/reconciler/annotations.go

```diff
@@ -44,14 +44,23 @@ type deploymentAnnotations struct {
 	// This will be nil if Radius is not enabled for the Deployment.
 	Configuration *deploymentConfiguration
 
-	//ConfigurationHash is the hash of the user-provided configuration.
+	// ConfigurationHash is the hash of the user-provided configuration.
 	// This will be used to diff the configuration and determine if the Deployment needs to be updated.
 	ConfigurationHash string
 
 	// Status is the status of the Deployment (Radius related status).
 	Status *deploymentStatus
 }
 
+// Ways to disable Radius:
+// 1. "radapp.io/enabled" annotation is set to "false"
+// 2. "radapp.io/enabled" annotation is not set
+
+// Case 1: Radius was Enabled and now it is Disabled.
+// Case 2: Radius was Disabled and now it is Enabled.
+// Case 3: Radius was and is Enabled.
+// Case 4: Radius was and is Disabled.
```

Comment:
This information is still missing context, it feels a lot more like personal notes than something that will be useful in the future. Someone who was present on the call where we discussed this might get it, but someone reading this in the future definitely won't. 


Can you try adding a few more sentences of explanation?